Makefile

```diff
@@ -2,7 +2,7 @@ CWD = $(shell pwd)
 SRC_DIR = ${CWD}/pootle
 DOCS_DIR = ${CWD}/docs
 STATIC_DIR = ${SRC_DIR}/static
-ASSETS_DIR = $(shell python -c "from pootle.settings import *; print(STATIC_ROOT)")
+ASSETS_DIR = ${SRC_DIR}/assets
```

Comment:
The `make assets` command is an optional helper, and hence it doesn't exist from the settings' standpoint. So I don't think any reference to the command belong in the settings file and in fact I find it rather confusing. As said a simple mention that the hard-coded value refers to the settings value would suffice.